stable/jupyter/templates/jupyter-configmap.yaml

```diff
@@ -53,18 +56,18 @@ data:
     ALIGN_MLRUN_SCRIPT="${HOME}/align_mlrun.sh"
     if [ ! -e ${ALIGN_MLRUN_SCRIPT} ]; then
       touch ${ALIGN_MLRUN_SCRIPT}
-      cat << EOF > ${ALIGN_MLRUN_SCRIPT}
-    CLIENT_MLRUN_VERSION=\`pip show mlrun | grep Version | awk '{print \$2}'\`
-    SERVER_MLRUN_VERSION=\`curl -s \${IGZ_MLRUN_API_ENDPOINT}/api/healthz | python3 -c "import sys, json; print(json.load(sys.stdin)['version'])"\`
-    if [ "\${CLIENT_MLRUN_VERSION}" = "\${SERVER_MLRUN_VERSION}" ] || [ "\${CLIENT_MLRUN_VERSION}" = "\${SERVER_MLRUN_VERSION//-}" ]; then
-      echo "Both server & client are aligned (\${CLIENT_MLRUN_VERSION})."
-    else
-      if [ \${CLIENT_MLRUN_VERSION} ]; then
-        pip uninstall -y mlrun
-      fi
-      pip install mlrun[complete]==\${SERVER_MLRUN_VERSION}
-    fi
-    EOF
+      echo "CLIENT_MLRUN_VERSION=\`pip show mlrun | grep Version | awk '{print \$2}'\`" >> ${ALIGN_MLRUN_SCRIPT}
```

Comment:
why not using @AlonMaor14 way of `cat` and inline bash commands without need to add eco everywhere? that way indentation is much clearer

Reply:
Just wanted to leave it the same as other parts of the configmap where echo is being used.
In some places there are for loops generating lines so cat is not an option

Reply:
I try and replace most of the config map to the cat way but let's do it in a later commit